CHANGELOG.md

```diff
@@ -28,6 +28,13 @@
   union other_employees
   ```
 
+- _Experimental:_ Excluding columns (@aljazerzen, #1329)
+
+  ```prql
+  from employees
+  select ![title, composer]
+  ```
```

Comment:
(it's worth putting the result here as it doesn't compile here, but I can do that when releasing)